crates/encoding/src/estimate.rs

```diff
@@ -288,22 +293,22 @@ fn transform_scale(t: Option<&Transform>) -> f32 {
     }
 }
 
-fn approx_arc_length_cubic(p0: Vec2, p1: Vec2, p2: Vec2, p3: Vec2) -> f64 {
+fn approx_arc_length_cubic(p0: Vec2, p1: Vec2, p2: Vec2, p3: Vec2) -> f32 {
```

Comment:
My gut feeling would be to keep these f64 to emphasize that they're CPU-side computations and to have as much accuracy as possible, but I don't think it makes a difference in practice. The later ones where we defer quantization are a good idea for sure though.